INFO - 2024-06-09 09:03:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:04:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:04:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:05:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-09 09:05:04 --> ErrorException: Could not check compatibility between Myth\Auth\Authentication\Passwords\CompositionValidator::check(string $password, ?CodeIgniter\Entity\Entity $user = null): bool and Myth\Auth\Authentication\Passwords\ValidatorInterface::check(string $password, ?CodeIgniter\Entity $user = null): bool, because class CodeIgniter\Entity is not available
[Method: POST, Route: register]
in APPPATH\ThirdParty\myth-auth\src\Authentication\Passwords\CompositionValidator.php on line 28.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-06-09 09:05:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:05:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-09 09:05:43 --> ErrorException: Could not check compatibility between Myth\Auth\Authentication\Passwords\CompositionValidator::check(string $password, ?CodeIgniter\Entity\Entity $user = null): bool and Myth\Auth\Authentication\Passwords\ValidatorInterface::check(string $password, ?CodeIgniter\Entity $user = null): bool, because class CodeIgniter\Entity is not available
[Method: POST, Route: register]
in APPPATH\ThirdParty\myth-auth\src\Authentication\Passwords\CompositionValidator.php on line 28.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-06-09 09:06:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:07:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:07:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:08:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-09 09:08:38 --> ErrorException: Could not check compatibility between Myth\Auth\Authentication\Passwords\CompositionValidator::check(string $password, ?CodeIgniter\Entity\Entity $user = null): bool and Myth\Auth\Authentication\Passwords\ValidatorInterface::check(string $password, ?CodeIgniter\Entity $user = null): bool, because class CodeIgniter\Entity is not available
[Method: POST, Route: register]
in APPPATH\ThirdParty\myth-auth\src\Authentication\Passwords\CompositionValidator.php on line 28.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-06-09 09:14:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-09 09:14:41 --> Error: Class name must be a valid object or a string
[Method: POST, Route: register]
in APPPATH\ThirdParty\myth-auth\src\Config\Services.php on line 81.
 1 SYSTEMPATH\Config\BaseService.php(311): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
 2 SYSTEMPATH\Config\BaseService.php(250): CodeIgniter\Config\BaseService::__callStatic('activator', [...])
 3 APPPATH\ThirdParty\myth-auth\src\Config\Services.php(74): CodeIgniter\Config\BaseService::getSharedInstance('activator', null)
 4 SYSTEMPATH\Config\BaseService.php(320): Myth\Auth\Config\Services::activator()
 5 SYSTEMPATH\Config\BaseService.php(201): CodeIgniter\Config\BaseService::__callStatic('activator', [])
 6 SYSTEMPATH\Common.php(998): CodeIgniter\Config\BaseService::get('activator')
 7 APPPATH\ThirdParty\myth-auth\src\Controllers\AuthController.php(235): service('activator')
 8 SYSTEMPATH\CodeIgniter.php(933): Myth\Auth\Controllers\AuthController->attemptRegister()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\basic-CI4\\public\\index.php')
INFO - 2024-06-09 09:14:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:15:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:15:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:15:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-09 09:15:49 --> Error: Class name must be a valid object or a string
[Method: GET, Route: resend-activate-account]
in APPPATH\ThirdParty\myth-auth\src\Config\Services.php on line 81.
 1 SYSTEMPATH\Config\BaseService.php(311): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
 2 SYSTEMPATH\Config\BaseService.php(250): CodeIgniter\Config\BaseService::__callStatic('activator', [...])
 3 APPPATH\ThirdParty\myth-auth\src\Config\Services.php(74): CodeIgniter\Config\BaseService::getSharedInstance('activator', null)
 4 SYSTEMPATH\Config\BaseService.php(320): Myth\Auth\Config\Services::activator()
 5 SYSTEMPATH\Config\BaseService.php(201): CodeIgniter\Config\BaseService::__callStatic('activator', [])
 6 SYSTEMPATH\Common.php(998): CodeIgniter\Config\BaseService::get('activator')
 7 APPPATH\ThirdParty\myth-auth\src\Controllers\AuthController.php(501): service('activator')
 8 SYSTEMPATH\CodeIgniter.php(933): Myth\Auth\Controllers\AuthController->resendActivateAccount()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\basic-CI4\\public\\index.php')
INFO - 2024-06-09 09:17:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:17:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:17:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:17:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:17:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:17:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:18:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:18:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:21:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-09 09:21:08 --> TypeError: App\Controllers\Home::home(): Return value must be of type string, none returned
[Method: GET, Route: home]
in APPPATH\Controllers\Home.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->home()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\basic-CI4\\public\\index.php')
INFO - 2024-06-09 09:22:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:25:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:25:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:25:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:25:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:33:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:33:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:33:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:33:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:33:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:34:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:41:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:41:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:41:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:41:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:41:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:41:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:44:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:44:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:44:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:44:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:45:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:57:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:57:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 09:57:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-09 10:01:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-09 10:01:36 --> mysqli_sql_exception: Table 'db_belajar_ci4.users' doesn't exist in C:\xampp\htdocs\basic-CI4\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\basic-CI4\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\basic-CI4\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\basic-CI4\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\basic-CI4\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\basic-CI4\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\basic-CI4\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp\htdocs\basic-CI4\app\ThirdParty\myth-auth\src\Authentication\AuthenticationBase.php(131): CodeIgniter\BaseModel->find('1')
#7 C:\xampp\htdocs\basic-CI4\app\ThirdParty\myth-auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp\htdocs\basic-CI4\app\ThirdParty\myth-auth\src\Controllers\AuthController.php(142): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp\htdocs\basic-CI4\system\CodeIgniter.php(933): Myth\Auth\Controllers\AuthController->logout()
#10 C:\xampp\htdocs\basic-CI4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp\htdocs\basic-CI4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\basic-CI4\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\basic-CI4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\basic-CI4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\basic-CI4\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2024-06-09 10:01:36 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_belajar_ci4.users' doesn't exist
[Method: GET, Route: logout]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = :users.id:', [...], false)
 2 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\ThirdParty\myth-auth\src\Authentication\AuthenticationBase.php(131): CodeIgniter\BaseModel->find('1')
 5 APPPATH\ThirdParty\myth-auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 6 APPPATH\ThirdParty\myth-auth\src\Controllers\AuthController.php(142): Myth\Auth\Authentication\LocalAuthenticator->check()
 7 SYSTEMPATH\CodeIgniter.php(933): Myth\Auth\Controllers\AuthController->logout()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\basic-CI4\\public\\index.php')
CRITICAL - 2024-06-09 10:01:36 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_belajar_ci4.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = \'1\'')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = \'1\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = :users.id:', [...], false)
 4 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
 6 APPPATH\ThirdParty\myth-auth\src\Authentication\AuthenticationBase.php(131): CodeIgniter\BaseModel->find('1')
 7 APPPATH\ThirdParty\myth-auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 8 APPPATH\ThirdParty\myth-auth\src\Controllers\AuthController.php(142): Myth\Auth\Authentication\LocalAuthenticator->check()
 9 SYSTEMPATH\CodeIgniter.php(933): Myth\Auth\Controllers\AuthController->logout()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\basic-CI4\\public\\index.php')
CRITICAL - 2024-06-09 10:01:36 --> [Caused by] mysqli_sql_exception: Table 'db_belajar_ci4.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = \'1\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = \'1\'')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = \'1\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = :users.id:', [...], false)
 5 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
 7 APPPATH\ThirdParty\myth-auth\src\Authentication\AuthenticationBase.php(131): CodeIgniter\BaseModel->find('1')
 8 APPPATH\ThirdParty\myth-auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 9 APPPATH\ThirdParty\myth-auth\src\Controllers\AuthController.php(142): Myth\Auth\Authentication\LocalAuthenticator->check()
10 SYSTEMPATH\CodeIgniter.php(933): Myth\Auth\Controllers\AuthController->logout()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\basic-CI4\\public\\index.php')
INFO - 2024-06-09 10:02:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-09 10:02:17 --> mysqli_sql_exception: Table 'db_belajar_ci4.users' doesn't exist in C:\xampp\htdocs\basic-CI4\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\basic-CI4\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\basic-CI4\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\basic-CI4\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\basic-CI4\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\basic-CI4\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\basic-CI4\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp\htdocs\basic-CI4\app\ThirdParty\myth-auth\src\Authentication\AuthenticationBase.php(131): CodeIgniter\BaseModel->find('1')
#7 C:\xampp\htdocs\basic-CI4\app\ThirdParty\myth-auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 C:\xampp\htdocs\basic-CI4\app\ThirdParty\myth-auth\src\Controllers\AuthController.php(62): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 C:\xampp\htdocs\basic-CI4\system\CodeIgniter.php(933): Myth\Auth\Controllers\AuthController->login()
#10 C:\xampp\htdocs\basic-CI4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp\htdocs\basic-CI4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\basic-CI4\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\basic-CI4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\basic-CI4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\basic-CI4\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2024-06-09 10:02:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_belajar_ci4.users' doesn't exist
[Method: GET, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = :users.id:', [...], false)
 2 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\ThirdParty\myth-auth\src\Authentication\AuthenticationBase.php(131): CodeIgniter\BaseModel->find('1')
 5 APPPATH\ThirdParty\myth-auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 6 APPPATH\ThirdParty\myth-auth\src\Controllers\AuthController.php(62): Myth\Auth\Authentication\LocalAuthenticator->check()
 7 SYSTEMPATH\CodeIgniter.php(933): Myth\Auth\Controllers\AuthController->login()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\basic-CI4\\public\\index.php')
CRITICAL - 2024-06-09 10:02:17 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'db_belajar_ci4.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = \'1\'')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = \'1\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = :users.id:', [...], false)
 4 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
 6 APPPATH\ThirdParty\myth-auth\src\Authentication\AuthenticationBase.php(131): CodeIgniter\BaseModel->find('1')
 7 APPPATH\ThirdParty\myth-auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 8 APPPATH\ThirdParty\myth-auth\src\Controllers\AuthController.php(62): Myth\Auth\Authentication\LocalAuthenticator->check()
 9 SYSTEMPATH\CodeIgniter.php(933): Myth\Auth\Controllers\AuthController->login()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\basic-CI4\\public\\index.php')
CRITICAL - 2024-06-09 10:02:17 --> [Caused by] mysqli_sql_exception: Table 'db_belajar_ci4.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = \'1\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = \'1\'')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = \'1\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = :users.id:', [...], false)
 5 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
 7 APPPATH\ThirdParty\myth-auth\src\Authentication\AuthenticationBase.php(131): CodeIgniter\BaseModel->find('1')
 8 APPPATH\ThirdParty\myth-auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 9 APPPATH\ThirdParty\myth-auth\src\Controllers\AuthController.php(62): Myth\Auth\Authentication\LocalAuthenticator->check()
10 SYSTEMPATH\CodeIgniter.php(933): Myth\Auth\Controllers\AuthController->login()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\basic-CI4\\public\\index.php')
